build.gradle

```diff
@@ -544,13 +563,6 @@ run {
   }
 }
 
-startScripts {
-  doLast {
-    unixScript.text = unixScript.text.replace('TEKU_HOME', '\$APP_HOME').replace('APP_NAME=', 'export LD_LIBRARY_PATH="\$APP_HOME/native:\$LD_LIBRARY_PATH"\nAPP_NAME=')
```

Comment:
We don't need this `LD_LIBRARY_PATH` line anymore?

Reply:
No, it was only for mothra which we dropped a while back. Sorry should have noted that.